.dockerignore

```diff
@@ -1,8 +1,29 @@
-.vscode
-.circleci
+# Hidden files and directories.
+.*
 
-vendor/
+# Top-level directories unrelated to the build.
+docs/
 infrastructure/
+scripts/
+staker-rewards/
 
-solidity/node_modules/
-solidity/build/
+# Top-level files unrealted to the build.
+CODEOWNERS
+crytic-config.json
+Dockerfile
+*.adoc
```

Comment:
This isn't really top-level, eh?

Reply:
With `.dockerignore` it will ignore only `*.adoc` files in the root directory. It works in a different way than `.gitignore` which would ignore files recursively. Was that your concern?

Reply:
Of course it works differently, why wouldn't it work differently? 🙃 Every once in a while I remember that software engineering is really just a game of how many slightly different models to do the exact same thing you can remember at the same time.

Reply:
🤷🏻‍♂️ 🤷🏻‍♀️